app/models/miq_alert.rb

```diff
@@ -434,7 +435,9 @@ def self.automate_expressions
         :options => [
           {:name => :mw_operator, :description => _("Operator"), :values => [">", ">=", "<", "<=", "="]},
           {:name => :value_mw_garbage_collector, :description => _("Duration Per Minute (ms)"), :numeric => true}
-        ]}
+        ]},
+      {:name => "hwk_generic", :description => _("All hawkular alerts"), :db => ["ContainerNode"], :responds_to_events => "datawarehouse_event",
```

Comment:
"All hawkular alerts" may need more context given the existing middleware conditions above?